src/arkhamdb/ArkhamDb.ttslua

```diff
@@ -0,0 +1,439 @@
+do
```

Comment:
Should this file maybe have `Api` in its name to be in line with the other ones?